diff --git a/dist/esm/generate-types.js b/dist/esm/generate-types.js
index d40624cc2fe5892f08bc054643afe97095fc81f3..1dbba1227fd99ecf1201f61d359244fc448bf9a6 100644
--- a/dist/esm/generate-types.js
+++ b/dist/esm/generate-types.js
@@ -233,7 +233,8 @@ export async function generateTypes(paths, config) {
             }
             const solidityOutput = parsedBuildInfo.output.contracts[parsed.inputSourceName][contractName];
             const hardhatArtifactObject = { ...parsed, ...solidityOutput };
-            const { buildInfoId, _format, ...artifactObject } = hardhatArtifactObject;
+            const { buildInfoId, _format, evm: _evm, metadata: _metadata, ...artifactObject } = hardhatArtifactObject;
+            // console.log(Object.keys(artifactObject));
             const fullName = `${dirname}/${contractName}`;
             allArtifacts[fullName] = artifactObject;
             if (shortNameDict[contractName]) {
