diff --git a/dist/executor/index.js b/dist/executor/index.js
index 54c273e03c2dd2896eb16dfa25eb222a531fc0c8..0d56d1c17657d8e75db857d9dd20bf25e664be1e 100644
--- a/dist/executor/index.js
+++ b/dist/executor/index.js
@@ -6,7 +6,8 @@ import { logger, setLogLevel, spin } from '../internal/logging.js';
 import { getRoughGasPriceEstimate } from '../utils/eth.js';
 import prompts from 'prompts';
 import { formatEther } from 'viem';
-import { tsImport } from 'tsx/esm/api';
+import { tsImport as tsImport_ } from 'tsx/esm/api';
+const tsImport = (path, opts) => typeof Bun !== 'undefined' ? import(path) : tsImport_(path, opts);
 export function execute(callback, options) {
     const scriptModule = (env, args) => callback(env, args);
     scriptModule.tags = options.tags;
